src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/OrganizationListPage.xaml

```diff
@@ -1,30 +1,30 @@
 ﻿<Page x:Class="Uno.AzureDevOps.Views.Content.OrganizationListPage"
 	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:local="using:Uno.AzureDevOps.Views.Content"
 	  xmlns:uc="using:Uno.AzureDevOps.Views.Controls"
-	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:toolkit="using:Uno.UI.Toolkit"
-	  xmlns:wasm="http:/uno.ui/wasm"
-	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 	  xmlns:beh="using:Uno.AzureDevOps.Views.Behaviors"
 	  xmlns:controls="using:Uno.AzureDevOps.Views.Controls"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:wasm="http:/uno.ui/wasm"
 	  mc:Ignorable="wasm">
 
 	<Page.Resources>
 		<DataTemplate x:Key="OrganizationListItem">
-			<Border HorizontalAlignment="Center"
+			<Border CornerRadius="10"
+					HorizontalAlignment="Center"
 					VerticalAlignment="Top"
```

Comment:
indentation

Reply:
Yes like we will do with Ctrl-K-D (with Tools/Options/TextEditor/XAML/Formatting/Spacing/"Position first attribute on same line as start tag" enabled)